t/cache-driver-file.t

```diff
@@ -9,12 +9,18 @@ my $ua     = Mojo::UserAgent->with_roles('+Cache')->new;
 my $driver = $ua->cache_driver;
 is $driver, $ua->cache_driver_singleton, 'using cache_driver_singleton';
 
-is $driver->get($url), undef, 'get';
+is $driver->get([post => $url]), undef, 'get';
 
-is $driver->set($url, "GET /file\r\n"), $driver, 'set';
-is_deeply $driver->get($url), "GET /file\r\n", 'get after set';
+is $driver->set([post => $url], "GET /file\r\n"), $driver, 'set';
+is_deeply $driver->get([post => $url]), "GET /file\r\n", 'get after set';
+like +Mojo::File->new($driver->root_dir)->list_tree->first, qr{mojo-useragent-cache-.*post[^\w]+cba7c48ddec7\.http},
+  'filename on disk';
 
-is $driver->remove($url), $driver, 'remove';
-is $driver->get($url), undef, 'get after remove';
+is $driver->remove([post => $url]), $driver, 'remove';
+is $driver->get([post => $url]), undef, 'get after remove';
+
+is $driver->set([get => $url], "GET /file\r\n"), $driver, 'set';
+like +Mojo::File->new($driver->root_dir)->list_tree->first, qr{mojo-useragent-cache-.*get[^\w]+cba7c48ddec7\.http},
+  'filename on disk';
```

Comment:
is it possible to test more of the host/path part here?

Reply:
Yes... Give me a sec.